CITATION.cff

```diff
@@ -0,0 +1,18 @@
+# file: CITATION.cff
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Yadan"
+  given-names: "Omry"
+  orcid: "https://orcid.org/0000-0002-1871-7216"
+- family-names: "Sommer-Simpson"
+  given-names: "Jasha"
+  orcid: "https://orcid.org/0000-0002-1397-6454"
+- family-names: "Delalleau"
+  given-names: "Olivier"
+  orcid: "https://orcid.org/0000-0002-0610-7226"
+title: "omegaconf"
+#version: 2.1.1
+#doi: 10.5281/zenodo.1234
+#date-released: 2021-08-17
```

Comment:
```suggestion
#doi: 10.5281/zenodo.1234
date-released: 2019-11-19
```
Omitting version and using the original release date.

Reply:
@Jasha10 I applied your suggested changes. However, @omry had initially suggested to drop the `date-released` field.

Reply:
Thanks @sugatoray,
The suggested change was based on [this comment](https://github.com/omry/omegaconf/issues/782#issuecomment-905847825) from the linked issue:
> "Can we just omit the version and have the original release date?"